auth/index.js

```diff
@@ -17,6 +17,7 @@ exports.middleware = function authMiddleware(req, res, next) {
     const user = await handler.isLoggedIn(req)
     if (user) {
       req.uid = user._id
+      req.ucollection = user.collection
```

Comment:
so if you look carefully its actually `ucollection` with a `u`, and i followed pretty much the same set of rules you followed for tacking on `uid` onto the `req`.

1. I added it onto the JWT so when a request with a token comes in, the `uid` and `ucollection` come with it. So the result from `isLoggedIn` will have it on it (if token is valid)
2. The other case in during login - but during login we know the collection because it is now set during route setup

I now see though how it can be confusing because other properties `_id` and `email` we are used to seeing on the `users` doc - maybe when assigning to the token i can name it something else? `_collection` ?